openedx/features/partners/give2asia/views.py

```diff
@@ -1,7 +1,8 @@
 import analytics
-import datetime
 import json
-import logging
+
+from datetime import datetime
```

Comment:
Let's please not make a separate group for these imports, and keep them in the standard library group.